pom.xml

```diff
@@ -77,10 +77,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>io.dropwizard</groupId>
-            <artifactId>dropwizard-metrics</artifactId>
-            <version>0.9.2</version>
-            <scope>provided</scope>
+            <groupId>io.dropwizard.metrics</groupId>
+            <artifactId>metrics-core</artifactId>
+            <version>3.1.2</version>
```

Comment:
I wish I could find better documentation about that dependency. Sounds like using `provided` scope for the purpose of avoiding version bumping is not a common practice, so this seems fine as is.